test/metabase/lib/drill_thru/column_extract_test.cljc

```diff
@@ -378,97 +362,15 @@
          :custom-query   query-regex
          :expected       {:type         :drill-thru/column-extract
                           :display-name "Extract domain"
-                          :extractions  [{:key :email-domain, :display-name "Domain"}]}
-         :drill-args     ["email-domain"]
-         :expected-query {:stages [{:expressions [[:regex-match-first {:lib/expression-name "Domain"}
-                                                   [:field {} (meta/id :people :email)]
-                                                   (u/regex->str email->domain-regex)]]}]}}))
+                          :extractions  [{:key :domain, :display-name "Domain"}
+                                         {:key :host,   :display-name "Host"}]}
+         :drill-args     ["domain"]
+         :expected-query {:stages [{:expressions [[:domain {:lib/expression-name "Domain"}
+                                                   [:field {} (meta/id :people :email)]]]}]}}))
     (testing "when the database does not support :regex email extraction is not available"
       (lib.drill-thru.tu/test-drill-not-returned
         {:drill-type     :drill-thru/column-extract
          :click-type     :header
          :query-type     :unaggregated
          :column-name    "EMAIL"
          :custom-query   query-no-regex}))))
-
```

Comment:
These tests moved to `metabase.legacy-mbql.util-test`, since that's where the regexes moved to as well.